solutions/tenants/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The API key to use for IBM Cloud."
+  sensitive   = true
+}
+
+variable "provider_visibility" {
+  description = "Set the visibility value for the IBM terraform provider. Supported values are `public`, `private`, `public-and-private`. [Learn more](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/guides/custom-service-endpoints)."
+  type        = string
+  default     = "public"
+
+  validation {
+    condition     = contains(["public", "private", "public-and-private"], var.provider_visibility)
+    error_message = "Invalid visibility option. Allowed values are 'public', 'private', or 'public-and-private'."
+  }
+}
+
+variable "tenant_configuration" {
+
+  description = "List of tenants to be created for log routing"
+  type = list(object({
+    new_tenant_region = string
+    new_tenant_name   = string
```

Comment:
since we will not support existing tenant, you can remove the `new_` prefix from the above 2 inputs